packages/nx/src/internal-testing-utils/mock-load-nx-plugin.ts

```diff
@@ -0,0 +1,14 @@
+jest.mock('nx/src/project-graph/plugins/loader', () => ({
+  ...jest.requireActual('nx/src/project-graph/plugins/loader'),
+  loadNxPlugin: jest.fn().mockImplementation(() => {
+    return [
+      Promise.resolve({
+        createNodes: ['', () => {}],
+        createDependencies: () => [],
+        include: [],
+        exclude: [],
```

Comment:
These are all optional fields right? We could just have this return an empty object.